src/components/Navbar/navigationItems.tsx

```diff
@@ -130,7 +130,10 @@ export const desktopMenuItems = ({
     key: 'discord',
     label: (
       <Link href="https://discord.gg/wFTh4QnDzk">
-        <a className="flex cursor-pointer items-center font-medium text-black hover:opacity-70 dark:text-slate-100">{t`Discord`}</a>
+        <a
+          className="flex cursor-pointer items-center font-medium text-black hover:opacity-70 dark:text-slate-100"
+          onClick={() => trackFathomGoal('PKIBZYTS')}
```

Comment:
**suggestion** could we use the goal constant variable from the constants file here and throughout?

Reply:
I think we all good now! 🍻